distribution-spec/v1.0/gitlab/PRODUCT.yaml

```diff
@@ -0,0 +1,8 @@
+vendor: GitLab
+name: GitLab Container Registry
+version: 16.5
+website_url: https://gitlab.com
+documentation_url: https://docs.gitlab.com/ee/user/packages/container_registry/
+product_logo_url: https://about.gitlab.com/images/press/logo/svg/gitlab-logo-100.svg
+type: product
```

Comment:
**Question:** Looking at the preview (https://deploy-preview-100--oci-conformance.netlify.app), I see that this is listed under the `Hosted` category. However, it should also be listed under `Open Source`, as GitLab has both hosted (GitLab.com) and Open Source/self-managed versions. Do I need to pick a different `type`?

Reply:
Does not seem possible with the current implementation: https://github.com/opencontainers/oci-conformance/blob/46075b988086c238a7950eb0cfbf27ec43199b85/products-page-generator/main.go#L204C8-L204C8.